Makefile

```diff
@@ -5,7 +5,7 @@ TEST_COVERAGE_DIR := $(BIN_DIR)/test-coverage
 GO_PACKAGES ?= ./...
 GO_TESTS ?= ^.*$
 GO_CMD ?= go
-GO_TEST ?= $(GO_CMD) test -count=1 -coverprofile=$(TEST_COVERAGE_DIR)/coverage.out -covermode=count -run $(GO_TESTS)
```

Comment:
Results from integration tests, which account for a significant portion of coverage percentage for the operator, were previously not reported to Codecov. Changes to the Makefile & to the workflow order in the ci.yml file above make it so unit test & integration test coverage report to separate files, which are both reported to Codecov.